src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -64,6 +64,13 @@ public static void ImaginaryOne()
             VerifyMagnitudePhaseProperties(Complex.ImaginaryOne, 1, Math.PI / 2);
         }
 
+        [Fact]
+        public static void SqrtMinusOne()
+        {
+            // Sqrt(-1) should be I exactly, not just tollerably close.
```

Comment:
Removing comment, adding netfx skip. The line I added to Sqrt_TestData will test this relation via a "tolerably close" criterion for netfx.